grouping.js

```diff
@@ -204,38 +251,53 @@ Partitioner = {
 			userId = Meteor.userId();
 		} catch (error) {}
 
-		return userId && this.getUserGroup(userId);
+		return userId && await this.getUserGroupAsync(userId);
+	},
+
+	// Static method to backwards-compatibly
+	group() {
+		return Meteor.wrapAsync(this.groupAsync)();
 	},
 
 	bindGroup(groupId, func) {
 		return this._currentGroup.withValue(groupId, func);
 	},
 
-	bindUserGroup(userId, func) {
-		const groupId = Partitioner.getUserGroup(userId);
+	async bindUserGroupAsync(userId, func) {
+		const groupId = Partitioner.getUserGroupAsync(userId);
 
 		if (!groupId) {
 			Meteor._debug(`Dropping operation because ${userId} is not in a group`);
 			return;
 		}
 
-		return Partitioner.bindGroup(groupId, func);
+		return await Partitioner.bindGroup(groupId, func);
+	},
+
+	// Static method to backwards-compatibly
+	bindUserGroup(userId, func) {
+		return Meteor.wrapAsync(this.bindUserGroupAsync)(userId, func);
 	},
 
 	directOperation(func) {
 		return this._directOps.withValue(true, func);
 	},
 
+	async _isAdminAsync(_id) {
+		return !!(await Meteor.users._partitionerDirect.findOneAsync({_id, admin: true}, {fields: {_id: 1}}));
+	},
+
+	// Static method to backwards-compatibly
 	_isAdmin(_id) {
-		return !!Meteor.users._partitionerDirect.findOne({_id, admin: true}, {fields: {_id: 1}});
+		return Meteor.wrapAsync(this._isAdminAsync)(_id);
```

Comment:
@wildhart here is most important to return result instead of Promise, but can I trust wrapAsync, when on other places does not work as axpected?